soc/atmel/sam/sam3x/Kconfig.soc

```diff
@@ -0,0 +1,42 @@
+# Atmel SAM3X MCU series
+
+# Copyright (c) 2014-2015 Wind River Systems, Inc.
+# Copyright (c) 2016 Intel Corporation.
+# Copyright (c) 2017 Justin Watson
+# Copyright (c) 2024 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_SAM3X
+	bool
+	help
```

Comment:
i.e. here

Reply:
Ok, so the series now `selects` the family. Is it not a `depends on`, right?

Reply:
Hi @nordicjm ,

Did you had time to check and confirm this?

Reply:
> Ok, so the series now `selects` the family. Is it not a `depends on`, right?

correct.